examples/tariff_7_first_hour_kwh_free.json

```diff
@@ -1,54 +1,71 @@
 {
   "country_code": "DE",
   "party_id": "ALL",
-  "id" : "52",
-  "currency" : "EUR",
-  "elements" : [ {
-    "price_components" : [ {
-      "type" : "PARKING_TIME",
-      "price" : 0.0,
-      "step_size" : 60
-    } ],
-    "restrictions" : {
-      "min_duration" : 0,
-      "max_duration" : 3600
+  "id": "52",
```

Comment:
Also in this example, most of this is only a layout change.

Only thing added to the file is: "last_updated": "2018-12-29T15:55:58Z"
Which is a good find.

Reply:
Added and fixed the date from month 17 to 12.